### PR TITLE
dxGantt: Sorting

### PR DESCRIPTION
**Pull request by AnnaKondratova**

### CODE REVIEW

File: api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md [hunk 1]
```diff
@@ -1,10 +1,182 @@
 ---
 id: dxGantt.Options.sorting
 type: dxGanttSorting
+inheritsType: dxGanttSorting
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Configures sorting settings.
```
Comment (AnnaGubareva) [inline api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md]: I'd remove '-ing' in this phrase. The option name contains 'sorting', but I'd try to use 'sort' in text where possible

```suggestion
Configures sort settings.
```

File: api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md [hunk 2]
```diff
@@ -1,10 +1,182 @@
...
+
+---
+
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Gantt/Sorting/"
+}
+
+Users can sort Gantt data by a single or multiple columns. Use the [mode](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode) option to specify the sort mode.
```
Comment (AnnaGubareva) [inline api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md:16]: 'mode' does not require an article

```suggestion
Users can sort Gantt data by a single or multiple columns. Use the [mode](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode) option to specify sort mode. 
```

File: api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md [hunk 3]
```diff
@@ -1,10 +1,182 @@
...
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Gantt/Sorting/"
+}
+
+Users can sort Gantt data by a single or multiple columns. Use the [mode](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode) option to specify the sort mode. 
+
+- **Single Mode**. Click a column header to sort data by this column. Subsequent clicks on the same header reverse the sort order.
+
+    ![DevExtreme Gantt - Sorting](/images/Gantt/sorting-single.png)
+
+- **Multiple Mode**. Hold Shift and click a column header to sort data by multiple columns. Subsequent clicks on the same header reverse the sort order and clear sorting for the column. Set the [showSortIndexes](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#showSortIndexes) option to **true** to show a column sort index within the header.
```
Comment (AnnaGubareva) [inline api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md]: You write 'a column header', but 'by multiple columns' (singular vs plural). Maybe you can write as follows:
"Click column headers with the Shift key pressed to sort data by multiple columns. "

'Subsequent clicks ...... clear sorting for the column' - It's unclear how subsequent clicks can both reverse sort order and clear sorting, Maybe you can give more details here. Should I use 'Shift' for subsequent clicks?

File: api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md [hunk 4]
```diff
@@ -1,10 +1,182 @@
...
+
+    ![DevExtreme Gantt - Sorting](/images/Gantt/sorting-multiple.png)
+
+[note] To disable sorting for a particular column, set the column’s **allowSorting** option to **false**.
```
Comment (AnnaGubareva) [inline api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md]: I don't know how notes are rendered on the resulting page, but maybe you can write this sentence as plain text without note formatting.

File: api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md [hunk 5]
```diff
@@ -1,10 +1,182 @@
...
+
+To clear sorting for a column, click Ctrl and the column header. You can also use the column header’s context menu to specify the column’s sort settings and clear sorting. Use the [ascendingText](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#ascendingText), [descendingText](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#descendingText) and the [clearText](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#clearText) options to specify texts for the corresponding context menu items.
```
Comment (AnnaGubareva) [inline api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md]: 1) `click Ctrl` -> `hold Ctrl`
2) Oxford comma

```suggestion
To clear sorting for a column, hold Ctrl and click the column header. You can also use the column header’s context menu to specify the column’s sort settings and clear sorting. Use the [ascendingText](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#ascendingText), [descendingText](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#descendingText), and the [clearText](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#clearText) options to specify text for the corresponding context menu items.
```

File: api-reference/_hidden/dxGanttSorting/mode.md [hunk 6]
```diff
@@ -6,7 +6,6 @@ default: 'single'
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies the sorting mode.
```
Comment (AnnaGubareva) [inline api-reference/_hidden/dxGanttSorting/mode.md]:
```suggestion
Specifies sort mode.
```

File: api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md [hunk 7]
```diff
@@ -1,10 +1,182 @@
...
-<!-- Description goes here -->
+Configures sort settings.
+
+---
+
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Gantt/Sorting/"
+}
+
+Users can sort Gantt data by a single or multiple columns. Use the [mode](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode) option to specify sort mode.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md]:
```suggestion
Users can sort Gantt data by a single or multiple columns. Use the [mode](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode) option to specify the sort mode. 
```

File: api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md [hunk 8]
```diff
@@ -1,10 +1,182 @@
...
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Gantt/Sorting/"
+}
+
+Users can sort Gantt data by a single or multiple columns. Use the [mode](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode) option to specify sort mode. 
+
+- **Single Mode**. Click a column header to sort data by this column. Subsequent clicks on the same header reverse the sort order.
+
+    ![DevExtreme Gantt - Sorting](/images/Gantt/sorting-single.png)
+
+- **Multiple Mode**. Hold Shift and click column headers to sort data by multiple columns. Subsequent clicks on the same header with the Shift key pressed reverse the column's sort order. Set the [showSortIndexes](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#showSortIndexes) option to **true** to show a column sort index within the header.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md]:
```suggestion
- **Multiple Mode**. Hold Shift and click column headers to sort data by multiple columns. Subsequent clicks on the same header with the Shift key pressed reverse the column's sort order. Set the [showSortIndexes](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#showSortIndexes) option to **true** to show the column sort index in the header.
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**